extension/src/tasks/taskUtil.ts

```diff
@@ -234,9 +234,10 @@ function createVSCodeTaskFromGradleTask(
         rootProject: gradleTask.getRootproject(),
         projectFolder: rootProject.getProjectUri().fsPath,
         workspaceFolder: rootProject.getWorkspaceFolder().uri.fsPath,
+        debugEnabled: gradleTask.getDebuggable(),
```

Comment:
why not simply call it 'debuggable' for consistency?

Reply:
Good idea. Done in https://github.com/microsoft/vscode-gradle/pull/1183/commits/88b07386df1cb4caf019dd7950856924fdb4f546